cmd/sonobuoy/app/gen_cli_docs.go

```diff
@@ -0,0 +1,24 @@
+package app
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+	"github.com/vmware-tanzu/sonobuoy/pkg/errlog"
+)
+
+func NewCmdGenCLIDocs() *cobra.Command {
+	var docsCmd = &cobra.Command{
```

Comment:
May be better to use := to maintain consistency, but this is just personal preference

Reply:
👍 I tend to prefer `:=` over `var...` as well; this was actually copied/modified from another command and I just didnt mind enough to change it. Its good that you're thinking of consistency/preference though. Just wasn't worth a new push/test/cycle.